.github/workflows/ci.yml

```diff
@@ -28,6 +29,11 @@ jobs:
             gemfile: rails7.0
           - ruby: '3.1'
             gemfile: rails7.1
+        exclude:
```

Comment:
Combining `include` and `exclude` makes the matrix hard to read (for humans, at least). I prefer to _only_ use `include`. E.g.

```
      matrix:
        ruby:
          - '3.2'
          - '3.3'
          - '3.4'
        gemfile:
          - rails7.1
          - rails7.2
          - rails8.0
          - rails_main
        include:
          - {ruby: '3.1', gemfile: rails6.1}
          - {ruby: '3.1', gemfile: rails7.0}
```

Reply:
Yeah as always a second Benjamin opinion in the Ruby realm :)